app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
     <string name="ctrl_consent">An app wants to capture your device traffic.\nDeny it if you are not sure!</string>
     <string name="deny">Deny</string>
     <string name="allow">Allow</string>
-    <string name="ctrl_consent_title">PCAPdroid Control Request</string>
+    <string name="ctrl_consent_title">PCAPdroid control request</string>
     <string name="ctrl_consent_denied">PCAPdroid: the control request was denied</string>
     <string name="ctrl_consent_allowed">PCAPdroid: the control request was allowed</string>
-    <string name="capture_interface">Capture Interface</string>
+    <string name="capture_interface">Capture interface</string>
     <string name="internet">Internet</string>
-    <string name="all_interfaces">All Interfaces</string>
+    <string name="all_interfaces">All interfaces</string>
     <string name="capturing_from">Capturing packets from \"%1$s\"</string>
-    <string name="edit_filter">Edit Filter</string>
-    <string name="edit_list">Edit List</string>
-    <string name="hidden_connections_rules">Hidden Connections Rules</string>
-    <string name="list_is_empty">List is empty</string>
+    <string name="edit_filter">Filter</string>
+    <string name="edit_list">Edit list</string>
+    <string name="hidden_connections_rules">Rules for hidden connections</string>
+    <string name="list_is_empty">The list is empty</string>
     <string name="malicious_connection_app">Malicious connection detected (%1$s)</string>
     <string name="show_only_malicious">Malicious connections</string>
     <string name="security">Security</string>
-    <string name="malware_detection">Malware Detection</string>
+    <string name="malware_detection">Malware detection</string>
     <string name="malware_detection_summary">Detect connections to known malicious hosts via third-party blacklists.</string>
     <string name="reset">Reset</string>
-    <string name="malware_whitelist">Malware Whitelist</string>
-    <string name="malware_whitelist_rules">Malware Whitelist Rules</string>
+    <string name="malware_whitelist">Malware whitelist</string>
+    <string name="malware_whitelist_rules">Rules for malware whitelisting</string>
     <string name="feature_not_available">This feature is currently unavailable for your device</string>
-    <string name="billing_connecting">The connection to Google Play is in progress, please retry in a while</string>
-    <string name="malware_whitelist_help">This user-defined whitelist is used to inhibit malware detection. Connections matching any of these rules will not be marked as malware. You can create rules by long pressing a malicious connection.</string>
-    <string name="hidden_connections_help">This list contains rules to hide connections from the connections list. Hiding connections allows you to focus on the relevant ones.</string>
+    <string name="billing_connecting">Trying to connect to Google Play… Please try again in a while.</string>
```

Comment:
Same string with different key exists

Reply:
Where?

Reply:
<string name="billing_failure">Could not connect to Google Play. Please try again in a while.</string>

Not quite sure if this one failed already though. Otherwise "try again" seems strange.

Reply:
I think "billing_connecting" could be removed, for now please leave unchanged